### DISCUSSION TITLE
最新内核编译固件内存泄漏严重

### DISCUSSION THREAD

**Discussion by ghost** [Boos4721/openwrt#128] [Bugs/问题反馈]
AX6，昨晚新编5.15内核固件，开机后内存持续减少，数小时降到100M+（改1G内存），之前一直稳定接近400m......
是某些补丁失效？？？
望大佬排查

**Comment by he110777** [comment]
我也是啊，内存很快就从40可用减到15，之后就自动重启。还有5.15的upnp好像一直有问题。

**Comment by ghost** [comment]
经测试新补丁貌似已修复内存泄漏，有待长时间验证......

**Reply by he110777** [reply]
是的，5月6日更新我编译好现在已经26小时没有重启了。但是upnp映射不了。

**Reply by ghost** [reply]
5.15的upnp的确无效，等修吧......
